Makefile

```diff
@@ -182,6 +184,11 @@ superuser: ## create a Django superuser
 	@$(COMPOSE_RUN_APP) python manage.py createsuperuser
 .PHONY: superuser
 
+prosody-admin: ## create prosody admin user
+	@echo "$(BOLD)Creating a prosody admin$(RESET)"
+	$(COMPOSE_RUN) prosody-app prosodyctl register admin prosody-app jm2bafdxJRzxR9SDJEEMsv2b2
```

Comment:
It would be great to reuse the `DJANGO_XMPP_PRIVATE_SERVER_PASSWORD` variable here to avoid duplication. Maybe by adding `include env.d/development` ?

Reply:
Done, can you take a look ?

Reply:
I would just add double quotes around it : 

```suggestion
$(COMPOSE_RUN) prosody-app prosodyctl register admin prosody-app "${DJANGO_XMPP_PRIVATE_SERVER_PASSWORD}"
```